modules/cnv-creating-bridge-nad-web.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * cnv-attaching-multiple-networks.adoc
+
+[id="cnv-creating-bridge-nad-web_{context}"]
+= Creating a Linux bridge NetworkAttachmentDefinition in the web console
+
+The NetworkAttachmentDefinition is a custom resource that exposes Layer 2 devices
+to a specific namespace in your {CNVProductName} cluster. 
+
+Network administrators can create NetworkAttachmentDefinitions 
+to provide existing Layer-2 networking to Pods and virtual machines. 
+
+.Prerequisites
+
+* A Linux bridge must be configured and attached to the correct
+Network Interface Card on every node.
+* If you use VLANs, `vlan_filtering` must be enabled on the bridge.
+* The NIC must be tagged to all relevant VLANs.
+** For example: `bridge vlan add dev bond0 vid 1-4095 master`
+
+.Procedure
+
+. In the {CNVProductName} console, click *Networking* -> *Network Attachments Definitions*.
+. Click *Create Network Attachment Definition* .
+. Enter a unique *Name* and optional *Description*.
+. Click the *Network Type* list and select *CNV Linux bridge*.
```

Comment:
The network type is named "CNV Linux bridge" in the UI? I wonder if we could request a change to "Container-native virtualization Linux bridge" -- is that too long?

Reply:
Raised it with Tomas. He reckons the full name is too long.